public/app/plugins/datasource/cloudwatch/components/MetricStatEditor/MetricStatEditor.test.tsx

```diff
@@ -40,9 +40,24 @@ describe('MetricStatEditor', () => {
     config.featureToggles.cloudWatchCrossAccountQuerying = originalFeatureToggleValue;
   });
   describe('statistics field', () => {
-    test.each(['Average', 'p23.23', 'p34', '$statistic'])('should accept valid values', async (statistic) => {
+    test.each([
+      'Average',
+      'p23.23',
+      'p34',
+      'TM(2%:98%)',
+      'WM(10%:90%)',
+      'tc99',
+      'ts90',
+      'TS(80%:)',
+      'PR(100:2000)',
+      'TC(0.005:0.030)',
+      'TM(100:200)',
+      'TM(:90%)',
+      'WM(45%:)',
+      '$statistic',
+    ])('should accept valid values', async (statistic) => {
       const onChange = jest.fn();
-
+      props.datasource.getVariables = jest.fn().mockReturnValue(['$statistic']);
```

Comment:
I think we can leave this test untouched, you can move your test cases to the other test (or if they are similar to the other entries you can choose which test cases to leave in)

Reply:
@sladyn98 this line 45 needs to be removed